### PR TITLE
[DOCS] Updating knowledge examples in taxonomy README.md

### PR DESCRIPTION
**Pull request by kelbrown20**
Description: Updating knowledge example in README per new SDG schema

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -282,44 +284,212 @@ Each `qna.yaml` file requires a minimum of five question-answer pairs. The `qna.
...
+          and three spiral galaxies NGC 88, NGC 89 and NGC 92.
+document_outline: |
+  Information about the Phoenix Constellation including the
+  history, characteristics, and features of the stars in the constellation.
 document:
-  repo: https://github.com/juliadenham/oscars2024_knowledge.git
-  commit: e1744af
-  patterns:
-    - oscars2024_results.md
+    repo: https://github.com/RedHatOfficial/rhelai-taxonomy-data
```
Comment (jjasghar) [inline README.md:482]: Do we want to link to the product here? should we create an "instructlab-example-taxonomy" or something repo for things like this?

**Review by jjasghar — CHANGES_REQUESTED** [review]
One question :)

File: README.md (hunk 1 above)
Comment (kelbrown20) [reply README.md:482]: Great point! I'll use the summit_bounty one

Comment (kelbrown20) [reply README.md]: Just updated that!

**Review by jjasghar — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (juliadenham) [reply README.md]: I knew I was forgetting something!! Yes it should be the summit repo info, will get that updated! Thanks JJ

**Review by jjasghar — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -334,32 +500,43 @@ This knowledge example references one markdown file: `oscars2024_results.md`. Yo
 What might these markdown files look like? They can be freeform. Here's what a
 snippet of `oscars2024_results.md` might look like in your Git repository.
```
Comment (juliadenham) [inline README.md:501]: Should be 'phoenix_constellation.md' instead of the oscars one!

**Review by juliadenham — COMMENTED** [review]
LGTM, just needs the markdown file name changed!

**Pull request merged**